sourcegraph/jetbrains
# Sa1to/implement cody chat messages history

It helps https://github.com/sourcegraph/sourcegraph/issues/54933

# Test plan:
* Open new cody chat window
* Send message 1
* Send message 2
* Press ⬆️, message 2 should appear in the prompt
* Press ⬆️, message 1 should appear in the prompt
* Press ⬇️, message 2 should again appear in the prompt
* Modify message 2 and send
* Press ⬆️, modified message 2 should appear
